pom.xml

```diff
@@ -254,6 +254,11 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>info.schnatterer.moby-names-generator</groupId>
```

Comment:
we've been trying to keep the dependencies ordered by groupId,artifactId within the group to which they belong.

Reply:
I'll move it, but again I ask what value does anyone derive from that?